codestodians/food-basket-api
# [Feature: #154589892] Reactivate an Inactive user account

#### What does this PR do?
- Enables User account reactivation.

#### Description of Task to be completed?
- Create mutation for reactivating a user account.
- Write test cases for different account reactivation scenarios.

#### How should this be manually tested?
- Check out into the `ft-user-account-reactivation-154589892` branch.
- Run the `reactivateUser` mutation on `graphiql` for: A. A fresh reactivation B. Reactivation for an already active account. C. Reactivation for a non-existent account.
- Ensure that you get the correct messages/results every time.
- Run tests with `coverage run --source=food_basket --omit */snapshots/* manage.py test`
- Ensure all lines are covered with `coverage report -m`

#### Any background context you want to provide?
- There should be the possibility of reactivation of a previously deactivated account so that persons who left the platform previously could continue using their previous account once reactivated without the hassle of creating a new account.

#### What are the relevant pivotal tracker stories?
- [#154589892](https://www.pivotaltracker.com/story/show/154589892)
